form.yml

```diff
@@ -232,7 +232,11 @@ hana:
         exposition_port:
           $name: SAP HANA exposter exposition port
           $type: text
-          $optional: true
+          $optional: false
+        multi_tenant:
+          $name: Enable Multi tenant monitoring
+          $type: boolean
```

Comment:
@arbulu89 if possible, I would include a `$help` here explaining that, if enabled, the exporter will select all the tenants listed on the SYSTEMDB metadata and open a connection to each of them in order to collect metrics...